assets/src/scripts/store.js

```diff
@@ -7,11 +9,23 @@ const { getMedianStatistics, getPreviousYearTimeseries, getTimeseries,
 const { normalize } = require('./schema');
 const { fetchFloodFeatures, fetchFloodExtent } = require('./floodData');
 
+const getLatestValue = function(collection, parmCd) {
+    let parmVar = findKey(collection.variables, (varValue) => {
+        return varValue.variableCode.value === parmCd;
+    });
+    let parmTimeSeries = findKey(collection.timeSeries, (ts) => {
+        return ts.variable === parmVar;
+    });
+    let points = parmTimeSeries ? collection.timeSeries[parmTimeSeries].points : [];
+    return points.length ? last(points).value : null;
+};
+
+const GAGE_HEIGHT_CD = '00065';
 
 export const Actions = {
-    retrieveTimeseries(siteno, params=null, startDate=null, endDate=null) {
```

Comment:
I removed these as parameters. The code at  line 35 and 36 which set the endTime and startTime assume that we have requested a period of time. This makes it explicit that this action gets the most recent data.